upi/vsphere/main.tf

```diff
@@ -29,7 +29,7 @@ module "bootstrap" {
 
   name             = "bootstrap"
   instance_count   = "${var.bootstrap_complete ? 0 : 1}"
-  ignition_url     = "${var.bootstrap_ignition_url}"
+  ignition         = "${file("${path.module}/ignition/bootstrap.ign")}"
```

Comment:
hmm, should we be using a variable for this? this makes the ignition in `{root}/ignition/bootstrap.ign` ??

who puts that file there?

Reply:
we should take a variable the points the file's location..

Reply:
What I was thinking was copying the ignition files to a certain location within the terraform directory path to reduce the quantity of variables that a user would need to modify.
Sure I could create a variable to hold the path.  Then in the terraform.tfvars.example use `${path.module}/ignition/node_type.ign`? Or do you not want to modify the directory structure within the `./upi/vsphere` ?

Reply:
Perhaps everything can be done inside of the same (upi/vsphere) directory? (openshift-install create ignition-configs + terraform apply) You could have variables passed into main.yaml with default path ${path.module}/node_type.ign.

Reply:
personally, i would like users to provide the filepath, and fail if they don't.
Because the installer generates all the ign files in a directory and the terraform {root} might be in a separate one, it makes sense to ask users to give us the location.

I'm fine if we think the user can point to the directory of the install where all the assets are, but explicit paths are more sustainable.